src/routes/blog/+page.svelte

```diff
@@ -3,10 +3,47 @@
     import { MainFooter, FooterNav, Article } from '$lib/components';
     import { TITLE_SUFFIX } from '$routes/titles.js';
     import { DEFAULT_HOST } from '$lib/utils/metadata';
+    import { _PAGE_NAVIGATION_RANGE } from '$routes/blog/+page';
+    import { page } from '$app/stores';
+    import { onMount } from 'svelte';
 
     export let data;
+    const featured = data.featured;
 
-    const featured = data.posts.find((post) => post.featured);
+    let currentPage = 1;
+
+    $: currentPageRange = (): number[] => {
+        const chunkIndex = Math.floor((currentPage - 1) / _PAGE_NAVIGATION_RANGE);
+        return data.navigation[chunkIndex] || [];
+    };
+
+    $: isLastPage = (): boolean => {
+        return currentPage < data.totalPages;
+    };
```

Comment:
We don't need a function here 
`isLastPage = currentPage < data.totalPages` should be enough

Reply:
Done.